portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteApps.testcase

```diff
@@ -693,15 +693,11 @@ definition {
 			RemoteApps.addType(type = "Custom Element");
 		}
 
-		task ("When type 'vanilla counter' in HTML Element field") {
-			Type(
-				key_text = "HTML Element Name",
-				locator1 = "TextInput#ANY",
-				value1 = "vanilla counter");
-		}
-
-		task ("When attempt to save the remote app") {
-			Button.clickPublish();
+		task ("When type 'vanilla counter' in HTML Element field and attempt to save the remote app") {
+			RemoteApps.addApp(
+				entryName = "RemoteApp Test Name",
+				entryURL = "$testurl",
```

Comment:
syntax ${} and camel case
${testURL}

Reply:
Hey John in this case and to fill in a url field. Can it have letters in uppercase?

Reply:
yes, can update to ${testURL}. I'm just matching what's existing in other tests ie: mainURL

Reply:
Ok, fixed! Thanks John 😃